### ISSUE TITLE
Compiling .tex is broken via miktex on Windows

### ISSUE THREAD

**Issue by ninohardt** [rstudio/rstudio#8843]
### System details

    RStudio Edition : Desktop
    RStudio Version :  1.4.1103
    OS Version      :  Windows 10
    R Version       :  4.0.2
    Tex:       pdfTeX, Version 3.14159265-2.6-1.40.21 (MiKTeX 21.1)

### Steps to reproduce the problem

`latex.tex` file:
```
\documentclass[12pt]{article}
\begin{document}

\section{Test}

This is a test

\end{document}
```
Hit 'compile PDF'

### Describe the problem in detail

Error message
```
Running pdflatex.exe on latex.tex...failed
Error running C:/PROGRA~1/MIKTEX~1.9/miktex/bin/x64/pdflatex.exe (exit code -1073740791)
```

### Describe the behavior you expected

It should just compile it

### Additional information

![image](https://user-images.githubusercontent.com/4387287/106216285-cb653000-61a0-11eb-8b01-5f7c3ee41156.png)

- [X] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [X] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [X] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kevinushey** [comment]
Unfortunately, I was not able to reproduce. My pdflatex version:

```
This is pdfTeX, Version 3.14159265-2.6-1.40.21 (MiKTeX 2.9.7380 64-bit) (preloaded format=pdflatex 2021.2.3)  3 FEB 2021 16:01
```

**Comment by ninohardt** [comment]
Just did a fresh installation of miktex on a clean Windows 10 VM.

> This is pdfTeX, Version 3.14159265-2.6-1.40.21 (MiKTeX 21.1)

Still does not work.

> Running pdflatex.exe on tex.tex...failed
> Error running C:/Users/user/AppData/Local/Programs/MiKTeX/miktex/bin/x64/pdflatex.exe (exit code -1073740791)

Maybe you rely on some system environment variables?
Try reproducing on a clean VM without you developer tools installed.

**Comment by benthegirl** [comment]
I am having an identical problem. I have tried so many ways to fix it. Here is a stackoverflow question I posted before I saw this bug report: [Error Running PdfLaTeX from R Studio Can't Compile PDFs since Updating](https://stackoverflow.com/questions/66285654/error-running-pdflatex-from-rstudio-cant-compile-pdfs-since-updating)

My post there has many details about what I'm running, what I've tried, and my system variables. Perhaps they will give some insight into this issue.
